lib/widgets/atoms/navigation_bar.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class CustomizedBottomNavigationBar extends StatefulWidget {
```

Comment:
Since this widget defines a `Scaffold` (along with e.g. top app bar) I'd consider renaming it to somethign else. Like `CustomizedScaffold` or `AppWrapper`.

Reply:
Makes sense, changed to AppWrapper